zilliqa/src/cfg.rs

```diff
@@ -366,7 +366,7 @@ pub struct ConsensusConfig {
     /// Forks in block execution logic. Each entry describes the difference in logic and the block height at which that
     /// difference applies.
     #[serde(default)]
-    pub forks: Forks,
+    pub forks: DeltaForks,
```

Comment:
How about we add `genesis_fork: Fork` which is always required and `forks: ForkDeltas` which can mutate those forks at higher blocks?

This way we can be sure that we always have a complete description of the execution behaviour and don't have to rely on the duplicate `Default` implementations we have here (which I think is error-prone).